src/layout/Group/GroupContainer.tsx

```diff
@@ -86,14 +84,16 @@ export function GroupContainer({ node }: IGroupProps): JSX.Element | null {
       {isLoading && (
         <AltinnLoader
           style={{ position: 'absolute' }}
-          srContent={`${getLanguageFromKey('general.add_new', language ?? {})} ${
-            resolvedTextBindings?.add_button ? getTextResourceByKey(resolvedTextBindings.add_button, textResources) : ''
-          }`}
+          srContent={
+            resolvedTextBindings?.custom_add_button
+              ? langAsString(resolvedTextBindings.custom_add_button)
+              : `${langAsString('general.add_new')} ${langAsString(resolvedTextBindings?.add_button) ?? ''}`
+          }
         />
       )}
-      {`${getLanguageFromKey('general.add_new', language ?? {})} ${
-        resolvedTextBindings?.add_button ? getTextResourceByKey(resolvedTextBindings.add_button, textResources) : ''
-      }`}
+      {resolvedTextBindings?.custom_add_button
+        ? langAsString(resolvedTextBindings.custom_add_button)
```

Comment:
Nice! Two nitpicks:
1. I don't think it has to be a string. We can get away with just using the `lang` function here, so that we support markdown, etc. Better to be flexible out of the box. The else-case here relies on string concatenation, so that still needs to be a string.
2. I'm not sure I like the `custom_add_button` name here. By definition, all text resources to you bind to a component are meant to set some custom value. How about `add_button_full` or something in that direction?

Reply:
Thx for the feedback, I have changed to lang for the if case, and kept langAsString in else case. 
I like the suggestion to change it to `add_button_full`, it makes more sense when you compare it with add_button in the docs.